File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        String[] ss = s.split("");
        String n = "0123456789";
        String[] nums = n.split("");
        int aa = s.length();
        if (aa == 4 || aa == 6) {
            for (int i = 0; i < aa; i++) {
                boolean flag = false;
                for (String what : nums) {
                    if (what.equals(ss[i])) {
                        flag = true;
                        break;
                    }
                }
                if (!flag) {
                    return false;
                }
            }
            return true;
            
        } else {
            return false;    
        }
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    const n = s.length;
    if (n === 4 || n === 6) {
        for (const w of s) {
            if (!"0123456789".includes(w)) {
                return false;
            }
        }
        return true;
    }
    else {
        return false;
    }
}